### PR TITLE
Fix unlinked docs and fail if unlinked

### LINKED ISSUE #2455: Pagination section does not expand in Kernels endpoint page

### LINKED ISSUE #2456: Documentation for linode/types does not expand pagination section

### LINKED ISSUE #2457: Documentation for domains/:id/records does not expand pagination section

### LINKED ISSUE #2458: Documentation for nodebalancers/:id/configs/:id/nodes does not expand pagination section, and has overflowing table cells

### LINKED ISSUE #2460: Documentation for GET support/tickets/:id is missing everything

### LINKED ISSUE #2461: Documentation for support/tickets/:id/replies is not rendering properly

### PR DESCRIPTION
**Pull request by eatonphil**
Closes #2458, closes #2460, closes #2461, closes #2455, closes #2456, closes #2457

### CODE REVIEW

File: docs/prebuild.js [hunk 1]
```diff
@@ -16,421 +16,436 @@ const pythonFiles = fs.readdirSync(pythonPath);
 const objectsPath = path.join(BASE_PATH, 'objects');
 const apiObjectMap = {};
 fs.readdirSync(objectsPath).filter(function(fileName) {
-  return path.extname(fileName) === '.yaml';
+    return path.extname(fileName) === '.yaml';
 }).forEach(function(fileName) {
-  const filePath = path.join(objectsPath, fileName);
-  apiObjectMap[fileName.split('.')[0].toLowerCase()] = yaml.safeLoad(fs.readFileSync(filePath, 'utf-8'), { json: true });
+    const filePath = path.join(objectsPath, fileName);
+    apiObjectMap[fileName.split('.')[0].toLowerCase()] = yaml.safeLoad(fs.readFileSync(filePath, 'utf-8'), { json: true });
```
Comment (na3d) [inline docs/prebuild.js]: why 4 spaces instead of 2?

Comment (eatonphil) [reply docs/prebuild.js]: Yeah wait a second... That's not the default.

Comment (eatonphil) [reply docs/prebuild.js]: Updated, thanks for the sanity check.

Comment (na3d) [reply docs/prebuild.js]: Yeah, I didn't know if there was a reason for it or not.  LGTM once it gives me the ability to merge.

**Comment by eatonphil** [comment]
This is going as a patch into master, squash and merge when ready please @na3d @displague

**Pull request merged**